src/.vs/config/applicationhost.config

```diff
@@ -0,0 +1,1030 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Well I am not sure how to do that. I deleted the file, and re-committed so it should be gone. I don't see it in the Files Changed section either. Are we good?